solrconfig.xml

```diff
@@ -237,6 +237,208 @@
+            http://wiki.apache.org/solr/LocalParams
+       -->
+
+       <str name="qf">
+         alt_names_t^100000.0
+         title_t^10000.0
+       </str>
+       <str name="pf">
+         alt_names_t^100000.0
+         title_t^10000.0
+       </str>
+
+       <int name="ps">3</int>
+       <float name="tie">0.01</float>
+
+       <!-- NOT using marc_display because it is large and will slow things down for search results -->
+       <str name="fl">
+         id,
+         format,
+         database_display,
+         title_t,
+         title_sort,
+         alt_names_t,
+         title_statement_display,
+         title_truncated_display,
+         az_vendor_id_display,
+         az_vendor_name_display,
+         note_display,
+         note_t,
+         availability_facet,
+         electronic_resource_display,
+         record_update_date,
+         subject_display,
+         subject_facet,
+         subject_t,
+         url_finding_aid_display:[json],
+         url_more_links_display:[json],
+         electronic_resource_display:[json]
+       </str>
+
+       <str name="facet">true</str>
+       <str name="facet.mincount">1</str>
+       <str name="facet.field">format</str>
+       <str name="facet.field">subject_facet</str>
+       <str name="facet.field">availability_facet</str>
+
+       <str name="spellcheck">false</str>
+
+     </lst>
+
+    <arr name="last-components">
+      <str>spellcheck</str>
+    </arr>
+
+  </requestHandler>
+
+  <requestHandler name="subject_search" class="solr.SearchHandler" default="true">
+    <!-- default values for query parameters can be specified, these
+         will be overridden by parameters in the request
+      -->
+     <lst name="defaults">
+       <str name="defType">edismax</str>
+       <str name="echoParams">explicit</str>
+       <str name="wt">json</str>
+       <int name="rows">10</int>
+
+       <str name="q.alt">*:*</str>
+       <str name="mm">5&lt;-1 8&lt;75%</str>
+       <str name="mm.autorelax">true</str>
+       <str name="lowercaseOperators">false</str>
+
+       <!-- this qf and pf are used by default, if not otherwise specified by
+            client. The default blacklight_config will use these for the
+            "keywords" search. See the author_qf/author_pf, title_qf, etc
+            below, which the default blacklight_config will specify for
+            those searches. You may also be interested in:
+            http://wiki.apache.org/solr/LocalParams
+       -->
+
+       <str name="qf">
+         subject_t^1000000.0
+         subject_facet^1000000.0
+       </str>
+       <str name="pf">
+         subject_t^1000000.0
+         subject_facet^1000000.0
+       </str>
+       <int name="ps">3</int>
+       <float name="tie">0.01</float>
+
+       <!-- NOT using marc_display because it is large and will slow things down for search results -->
+       <str name="fl">
+         id,
+         format,
+         database_display,
```

Comment:
Do we have a database_display field in the catalog?  I don't see it defined in the controller.

Reply:
Sorry, I just realized this IS the AZ solr!